src/store/index.js

```diff
@@ -1096,16 +1096,17 @@ export default createStore({
     },
     pages: state => {
       const arr = []
-      state.pages.forEach(page => {
-        const obj = {
-          tileSource: page.uri,
-          width: page.width,
-          x: 0,
-          y: 0
-        }
-        arr.push(obj)
-      })
-      return arr
+  state.pages.forEach(page => {
```

Comment:
I think the old indentation is correct?

Reply:
Sorry, maybe my comment was too brief. There are a lot of insignificant(?) whitespace changes here that make it harder to review. So, if the changes are irrelevant, then please remove them from the PR

Reply:
reverted in [6bcda40](https://github.com/Edirom/cartographer-app/pull/13/commits/6bcda4031fb334d69732e262d4f91ba08a28c821) -- thanks!